borsh/src/ser/mod.rs

```diff
@@ -210,6 +210,32 @@ impl BorshSerialize for String {
     }
 }
 
+#[cfg(feature = "ascii")]
+impl BorshSerialize for ascii::AsciiStr {
+    #[inline]
+    fn serialize<W: Write>(&self, writer: &mut W) -> Result<()> {
+        self.as_bytes().serialize(writer)
+    }
+}
+
+#[cfg(feature = "ascii")]
+impl BorshSerialize for ascii::AsciiString {
+    #[inline]
+    fn serialize<W: Write>(&self, writer: &mut W) -> Result<()> {
+        self.as_bytes().serialize(writer)
+    }
+}
+
+#[cfg(feature = "ascii")]
+#[test]
+fn test_ascii() {
+    let val = ascii::AsciiStr::from_ascii("foo").unwrap();
+    let encoded = borsh::to_vec(val).unwrap();
+
+    let got: String = borsh::BorshDeserialize::deserialize_reader(&mut &encoded[..]).unwrap();
+    assert_eq!("foo", got);
+}
```

Comment:
please add 2 lines to `.github/test.sh` to actually test this new feature (`std` and `no-std`),
similar to https://github.com/near/borsh-rs/blob/master/.github/test.sh#L10 and https://github.com/near/borsh-rs/blob/master/.github/test.sh#L16